core/trino-main/src/main/java/io/trino/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -318,6 +320,8 @@ private Set<QualifiedTablePrefix> calculatePrefixesWithTableName(
                         metadata.listViews(session, prefix).stream()))
                 .filter(objectName -> predicate.get().test(asFixedValues(objectName)))
                 .map(QualifiedObjectName::asQualifiedTablePrefix)
+                .distinct()
```

Comment:
Why `.distinct()` if we later collect to set?

add a comment (capture the purpose of this PR in a code comment)

Reply:
Thanks for the review. I will add the comments.

Reply:
@findepi Added comment stating the purpose in the first return statement. Thanks!